tests/QARegressionTests/test_core/test_test_command.py

```diff
@@ -28,6 +28,8 @@ def test_test_command_with_all_options(self):
                 "3.4",
                 "-dv",
                 os.path.join("tests", "resources", "report_test_data", "define.xml"),
+                "-dxp",

```

Comment:
Does this test fail without this argument? I don't think we want it to

Reply:
> Does this test fail without this argument?

No it doesn't fail without this argument. I'd add a separate test but assumed it was covered by the other tests in this file without the argument?